chips/cc2650/src/peripheral_interrupts.rs

```diff
@@ -0,0 +1,35 @@
+#[allow(non_camel_case_types)]
```

Comment:
Do you really need ```non_camel_case``` here (looks good to me)? But you might want to use an enum instead!

Reply:
Yep, `non_camel_case` is needed for names such as `AUX_PROG` etc. However, making it an enum is suitable; I'll change it :)

Reply:
Actually, using an `enum` here makes the pattern matching in chip.rs a bit more complicated, so lets stick with constants as of now